vue-frontend/src/pages/jobs/thank-you/index.vue

```diff
@@ -73,12 +73,15 @@ export default {
   },
   data() {
     return {
-      applicantName:
-        localStorage.getItem("applicant-name") != ""
-          ? JSON.parse(localStorage.getItem("applicant-name"))
-          : "Dear Applicant",
+      applicantName: "Dear Applicant",
```

Comment:
i think it should be null.

Reply:
I think its better to take this instead of null to prevent runtime errors